camera/android/src/main/java/com/capacitorjs/plugins/camera/CameraPlugin.java

```diff
@@ -268,19 +280,21 @@ public void openPhotos(final PluginCall call) {
         openPhotos(call, false, false);
     }
 
+    // FarmQA: Multiple photo selection is disabled.
+    // FarmQA: If/when enabled, the filename settings won't work.
     private void openPhotos(final PluginCall call, boolean multiple, boolean skipPermission) {
         if (skipPermission || checkPhotosPermissions(call)) {
             Intent intent = new Intent(Intent.ACTION_PICK);
             intent.putExtra(Intent.EXTRA_ALLOW_MULTIPLE, multiple);
             intent.setType("image/*");
             try {
-                if (multiple) {
-                    intent.putExtra("multi-pick", multiple);
-                    intent.putExtra(Intent.EXTRA_MIME_TYPES, new String[] { "image/*" });
-                    startActivityForResult(call, intent, "processPickedImages");
-                } else {
+//                if (multiple) {
```

Comment:
We could uncomment these lines since the multiple value is always false. It may make future pulls easier.